File: C#/bee_1013.cs
using System;

class URI
{
    static void Main(String[] args)
    {
       String [] numeros = Console.ReadLine().Split(' ');

        int a = int.Parse(numeros[0]);
        int b = int.Parse(numeros[1]);
        int c = int.Parse(numeros[2]);

        int maiorab = (a + b + Math.Abs(a - b))/2; 
        int maior = (maiorab + c + Math.Abs( maiorab - c))/2;

        Console.WriteLine($"{maior} eh o maior");
    }
}

File: Python/bee_1013.py
a,b,c = input().split()

a = int(a)
b = int(b)
c = int(c)

maiorab = (a + b + abs(a - b))/2
maior = (maiorab + c + abs(maiorab - c))/2

print(f"{maior :.0f} eh o maior")